yarn.lock

```diff
@@ -7400,6 +7509,13 @@ rollup@^2.77.2:
   optionalDependencies:
     fsevents "~2.3.2"
 
+rollup@^2.77.3:
```

Comment:
Before merging, we should check that this entry disappears as it should merge with the previous one. But it is not a blocker.

Reply:
Nope. Still hasn't disappeared.

Reply:
When I added rollup to the package.json file for each example, it set: `"rollup": "^2.77.3"`.

Reply:
Would you mind running `yarn why rollup` to find why we have two versions?

Reply:
Here is my console log for `yarn why rollup`:

```
yarn why rollup                                                                                                                                                                                                                                                                              rollup
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "rollup"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "rollup@2.77.2"
info Has been hoisted to "rollup"
info Reasons this module exists
   - "workspace-aggregator-511ef2db-8ac6-4e9e-8474-e5094676397f" depends on it
   - Hoisted from "_project_#@lumino#algorithm#rollup"
   - Hoisted from "_project_#@lumino#application#rollup"
   - Hoisted from "_project_#@lumino#collections#rollup"
   - Hoisted from "_project_#@lumino#commands#rollup"
   - Hoisted from "_project_#@lumino#coreutils#rollup"
   - Hoisted from "_project_#@lumino#datagrid#rollup"
   - Hoisted from "_project_#@lumino#disposable#rollup"
   - Hoisted from "_project_#@lumino#domutils#rollup"
   - Hoisted from "_project_#@lumino#dragdrop#rollup"
   - Hoisted from "_project_#@lumino#keyboard#rollup"
   - Hoisted from "_project_#@lumino#messaging#rollup"
   - Hoisted from "_project_#@lumino#polling#rollup"
   - Hoisted from "_project_#@lumino#properties#rollup"
   - Hoisted from "_project_#@lumino#signaling#rollup"
   - Hoisted from "_project_#@lumino#virtualdom#rollup"
   - Hoisted from "_project_#@lumino#widgets#rollup"
info Disk size without dependencies: "6.43MB"
info Disk size with unique dependencies: "6.43MB"
info Disk size with transitive dependencies: "6.43MB"
info Number of shared dependencies: 0
=> Found "@lumino/example-accordionpanel#rollup@2.77.3"
info This module exists because "_project_#@lumino#example-accordionpanel" depends on it.
info Disk size without dependencies: "6.43MB"
info Disk size with unique dependencies: "6.43MB"
info Disk size with transitive dependencies: "6.43MB"
info Number of shared dependencies: 0
=> Found "@lumino/example-datagrid#rollup@2.77.3"
info This module exists because "_project_#@lumino#example-datagrid" depends on it.
info Disk size without dependencies: "6.43MB"
info Disk size with unique dependencies: "6.43MB"
info Disk size with transitive dependencies: "6.43MB"
info Number of shared dependencies: 0
=> Found "@lumino/example-dockpanel#rollup@2.77.3"
info This module exists because "_project_#@lumino#example-dockpanel" depends on it.
info Disk size without dependencies: "6.43MB"
info Disk size with unique dependencies: "6.43MB"
info Disk size with transitive dependencies: "6.43MB"
info Number of shared dependencies: 0
✨  Done in 0.85s.
```